jwst/pipeline/calwebb_coron3.py

```diff
@@ -158,17 +166,22 @@ def process(self, user_input):
         try:
             completed = result.meta.cal_step.resample
         except AttributeError:
-            self.log.debug('Could not determine whether resample was completed. Presuming not.')
+            self.log.debug('Could not determine if resample was completed.')
+            self.log.debug('Presuming not.')
+
             completed = 'SKIPPED'
         if completed == 'COMPLETE':
             self.log.debug(f'Blending metadata for {result}')
             blendmeta.blendmodels(result, inputs=targ_files)
 
         try:
-            result.meta.asn.pool_name = asn['asn_pool']
+            result.meta.asn.pool_name = \
+                input_models.meta.\
```

Comment:
As in line 66 above, should be able to do

`result.meta.asn.pool_name = input_models.meta.asn_table.asn_pool`